sp/src/game/server/ai_speech_new.h

```diff
@@ -545,11 +545,11 @@ class CAI_ExpresserHostWithData : public CAI_ExpresserHost<BASE_NPC>
 protected:
 	EXPRESSER_TYPE *CreateExpresser( void )
 	{
-		AssertMsg1( m_pExpresser == NULL, "Tried to double-initialize expresser in %s\n", GetDebugName()  );
+		AssertMsg1( m_pExpresser == NULL, "Tried to double-initialize expresser in %s\n", this->GetDebugName()  );
```

Comment:
For clarification, the `this->` here is needed for calls to the base-class, because depending on the concrete template-parameter it is possible that the base-class does not provide the `GetDebugName()` (or 'Key()` below) methods. Writing `this->` tells the compiler that we **do** expect this class to provide these methods and call them here.